book/C-git-commands/1-git-commands.asc

```diff
@@ -118,64 +118,64 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
 ==== git difftool
 
-Ang `git difftool` na kautosan ay naglulunsad lamang ng isang panlabas na kasangkapan upang ipakita sa iyo ang pagkakaiba sa pagitan ng dalawang puno kung sakaling gusto mong gumamit ng iba pang bagay kaysa sa built in na `git diff` na kautusan.
+Ang `git difftool` na kautosan ay naglulunsad lamang ng isang panlabas na kasangkapan upang ipakita sa iyo ang pagkakaiba sa pagitan ng dalawang mga tree kung sakaling gusto mong gumamit ng iba pang bagay kaysa sa built in na `git diff` na kautusan.
 
 Binanggit lang namin ito sa <<_git_basics_chapter#_git_diff_staged>>.
 
 ==== git commit
 
-Ang `git commit` na kautusan ay kumukuha sa lahat ng mga nilalaman ng file na itinanghal sa `git add` at nagtatala ng isang bagong permanenteng snapshot sa database at pagkatapos ay inililipat ang branch pointer sa kasalukuyang branch hanggang dito.
+Ang `git commit` na kautusan ay kumukuha sa lahat ng mga nilalaman ng file na itinanghal sa `git add` at nagtatala ng isang bagong permanenteng snapshot sa database at pagkatapos ay inililipat ang branch pointer hangang sa kasalukuyang branch nito.
 
-Unang sinaklaw natin ang mga pangunahing kaalaman sa paggawa sa <<_git_basics_chapter#_committing_changes>>. Ipinakita din namin kung paano gamitin ang isang `-a` na bandila upang laktawan ang `git add` na hakbang sa araw-araw na daloy ng trabaho at kung paano gamitin ang `-m` flag upang pumasa ng isang gumawa ng mensahe sa sa command line sa halip ng pagpapaputok ng isang editor.
+Unang tinalakay natin ang mga pangunahing kaalaman sa pag-commit sa <<_git_basics_chapter#_committing_changes>>. Ipinakita din namin kung paano gamitin ang isang `-a` na palatandaan upang laktawan ang `git add` na hakbang sa pang-araw-araw na daloy ng trabaho at kung paano gamitin ang `-m` na palatandaan upang pumasa ng isang mensaheng commit sa command line sa halip ng pagpapatakbo nito sa editor.
 
-Sa _git_basics_chapter.asc tinatakpan namin ang paggamit ng --amam na opsyon upang gawing muli ang pinakabagong gawa.
+Sa _git_basics_chapter.asc tinalakay namin ang paggamit ng --amend na opsyon upang gawing muli ang pinakabagong commit.
 
-Sa _git_branching.asc, pumunta kami sa mas maraming detalye tungkol sa kung ano ang ginagawa ng git at kung bakit ginagawa nito ito.
+Sa _git_branching.asc, pumunta kami sa mas maraming detalye tungkol sa kung ano ang ginagawa ng git at kung bakit ganito niya ginagawa.
 
-Tiningnan namin kung paano mag-sign magawa ng cryptographically kasama ang `-S` flag sa <<_git_tools#_signing_commits>>.
+Tiningnan namin kung paano mag-sign ng mga commit na cryptographically kasama ang `-S` na palatandaan sa <<_git_tools#_signing_commits>>.
```

Comment:
i think flag is a reserve word on  git.  please check   :    flag    and    na palatandaan